compiler/rustc_middle/src/ty/adjustment.rs

```diff
@@ -25,10 +25,10 @@ pub enum PointerCoercion {
     ArrayToPointer,
 
     /// Unsize a pointer/reference value, e.g., `&[T; n]` to
-    /// `&[T]`. Note that the source could be a thin or fat pointer.
+    /// `&[T]`. Note that the source could be a thin or wide pointer.
     /// This will do things like convert thin pointers to fat
     /// pointers, or convert structs containing thin pointers to
-    /// structs containing fat pointers, or convert between fat
+    /// structs containing wide pointers, or convert between fat
```

Comment:
```suggestion
    /// structs containing wide pointers, or convert between wide
```

the search and replace should probably also check for "fat \n pointer"